Remotion/Data/DomainObjects/ClientTransactionFactory.cs

```diff
@@ -19,20 +19,25 @@
 namespace Remotion.Data.DomainObjects
 {
   /// <summary>The <see cref="ClientTransactionFactory"/> is the default implementation of the <see cref="ITransactionFactory"/> interface.</summary>
+  /// <threadsafety static="true" instance="true"/>
   [Serializable]
   public class ClientTransactionFactory : ITransactionFactory
   {
+    ITransaction ITransactionFactory.CreateRootTransaction ()
+    {
+      return CreateRootTransaction().ToITransaction();
+    }
+
```

Comment:
I think I usually put the explicit interface method after the public API (but before any protected virtual implementations). I can live with both.